bedrock/firefox/redirects.py

```diff
@@ -312,4 +314,58 @@ def firefox_mobile_faq(request, *args, **kwargs):
     redirect(r'^firefox/personal', 'firefox.new'),
     redirect(r'^firefox/upgrade', 'firefox.new'),
     redirect(r'^firefox/ie', 'firefox.new'),
+
+    # Bug 1255882
+    redirect(r'^projects/bonecho/anti-phishing/?$',
+             'https://support.mozilla.org/kb/how-does-phishing-and-malware-protection-work'),
+    redirect(r'^projects/bonecho(/.*)?$', '/firefox/channel/'),
+    redirect(r'^projects/bonsai(/.*)?$', 'https://wiki.mozilla.org/Bonsai'),
+    redirect(r'^projects/camino(/.*)?$', 'http://caminobrowser.org/'),
+    redirect(r'^projects/cck(/.*)?$', 'https://wiki.mozilla.org/CCK'),
+    redirect(r'^projects/chimera(/.*)?$', 'http://caminobrowser.org/'),
+    redirect(r'^projects/deerpark(/.*)?$', '/firefox/channel/'),
+    redirect(r'^projects/embedding(/.*)?$',
+             'https://developer.mozilla.org/en/Embedding_Mozilla'),
+    redirect(r'^projects/granparadiso(/.*)?$', '/firefox/channel/'),
+    redirect(r'^projects/inspector(/.*)?$',
+             'https://developer.mozilla.org/En/DOM_Inspector'),
+    redirect(r'^projects/javaconnect(/.*)?$',
+             'http://developer.mozilla.org/en/JavaXPCOM'),
+    redirect(r'^projects/marketing(/.*)?$',
+             'http://contribute.mozilla.org/Marketing'),
+    redirect(r'^projects/minefield(/.*)?$',
+             'http://www.mozilla.org/projects/firefox/prerelease.html'),
+    redirect(r'^projects/minimo(/.*)?$', 'https://wiki.mozilla.org/Mobile'),
+    redirect(r'^projects/namoroka(/.*)?$', '/firefox/channel/'),
+    redirect(r'^projects/nspr(?:/.*)?$', 'https://developer.mozilla.org/docs/NSPR'),
+    redirect(r'^projects/netlib(/.*)?$', 'https://developer.mozilla.org/en/Necko'),
```

Comment:
This works, but is redirected to https://developer.mozilla.org/en-US/docs/Mozilla/Projects/Necko